third_party/nvfuser/csrc/type.cpp

```
@@ -1014,6 +1014,14 @@ std::ostream& operator<<(std::ostream& out, const BinaryOpType botype) {
   return out << binary_op_type2string(botype);
 }
 
+std::ostream& operator<<(std::ostream& out, const ScatterOpType sotype) {
+  if (sotype == ScatterOpType::Set)
+    return out << "scatter";
+  if (sotype == ScatterOpType::Add)
+    return out << "scatter_add";
+  return out << "scatter_error";
```

Just assert and fail here rather than printing an error string.